packages/manager/apps/dedicated/client/app/dedicated/server/kvm/order/routing.js

```diff
@@ -0,0 +1,16 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicated-server.server.order-kvm', {
+    url: '/kvm/order',
+    views: {
+      'tabView@app.dedicated-server.server': {
+        component: 'dedicatedServerKvmOrder',
+      },
+    },
+    resolve: {
+      goBack: /* @ngInject */ (goToServerDetails) => goToServerDetails,
+      user: /* @ngInject */ (currentUser) => currentUser,
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('server_configuration_impi_order_kvm'),
```

Comment:
I have take a habit to put sort resolve like this:

1. variables
2. functions
3. handler
4. goTo functions

May be we can use that to be able to read quickly a resolve and binding declaration

Reply:
its in the same order right?